src/main/scala/com/itechart/ny_accidents/service/metric/PopulationMetricService.scala

```diff
@@ -4,6 +4,7 @@ import com.google.inject.Inject
 import com.itechart.ny_accidents.database.dao.PopulationStorage
 import com.itechart.ny_accidents.entity.MergedData
 import com.itechart.ny_accidents.service.PopulationService
+import com.itechart.ny_accidents.utils.NumberUtils
```

Comment:
Sounds like this import isn't used here

Reply:
https://github.com/itechartbigdatalab/NY-Accidents-Rate/pull/56/commits/0ca555bd973fed5eb36e11b9215abe2ce98024e1